oe_content.install

```diff
@@ -14,6 +14,10 @@ function oe_content_install($is_syncing) {
   // Configure the RDF SKOS graphs.
   if (!$is_syncing) {
     \Drupal::service('oe_content.op_skos_setup')->setup();
+    // Add 'EU political leader name' vocabulary.
```

Comment:
It was already added to `oe_content_sub_entity_author_install`. Why do we need it in oe_content?